stable/redis-ha/templates/redis-ha-configmap.yaml

```
@@ -51,7 +51,8 @@ data:
 
     sentinel_update() {
         echo "Updating sentinel config"
-        sed -i "1s/^/$(cat sentinel-id)\\n/" "$SENTINEL_CONF"
+        eval MY_SENTINEL_ID=\${SENTINEL_ID_$INDEX}
```

I forgot about the `shell check` tests on the scripts to keep them as posix sh compatible as possible. This is the output from the failing tests:
```
In /readonly-config/init.sh line 15:
    eval MY_SENTINEL_ID=\${SENTINEL_ID_$INDEX}
                          ^-- SC1083: This { is literal. Check expression (missing ;/\n?) or quote it.
                                       ^-- SC2086: Double quote to prevent globbing and word splitting.
                                             ^-- SC1083: This } is literal. Check expression (missing ;/\n?) or quote it.
```

---

basically this should do the trick:
```suggestion
        eval MY_SENTINEL_ID="\${SENTINEL_ID_$INDEX}"
```

---

Thanks, quoted to make shellcheck happy.

Normally I have shellcheck live linting in my editor, but spangled with braces and stuff it doesn't work ...